profiles/features/selinux/package.mask

```diff
@@ -34,3 +34,4 @@ net-misc/netctl
 sys-apps/gentoo-systemd-integration
 sys-apps/systemd-readahead
 sys-process/systemd-cron
+sys-apps/dbus-broker
```

Comment:
dbus-broker has selinux support:
https://news.ycombinator.com/item?id=15081662

Reply:
Unfortunately, our selinux profiles mask systemd, so this package must also be masked.